src/analysis/bounded_variation.lean

```diff
@@ -590,13 +591,13 @@ begin
 end
 
 lemma comp_le_of_monotone_on (f : α → E) {s : set α} {t : set β} (φ : β → α)
-  (hφ : monotone_on φ t) (φst : set.maps_to φ t s) :
+  (hφ : monotone_on φ t) (φst : t.maps_to φ s) :
```

Comment:
```suggestion
  (hφ : monotone_on φ t) (φst : maps_to φ t s) :
```
The `t.mapsto` stuff breaks common mathematical sense: it's not the set that maps anything. So better avoid dot notation, but `open set` to type `maps_to` instead of `set.maps_to`.

Reply:
Did the changes for `maps_to` and `surj_on` **and** `eq_on`, though I don't think this one is a cause for ire?